login-nl.po

```diff
@@ -94,7 +94,7 @@ msgstr "Vertrouw dit apparaat voor %{ days } dagen"
 
 #: src/js/login/components/two-step-auth.vue:3
 msgid "Two Step Authentication"
-msgstr "Tweestaps authenticatie"
+msgstr "Tweetraps authenticatie"
```

Comment:
Waarom heb je gekozen voor een `Tweetraps authenticatie`? Ik weet niet beter dan dat we dit ook in het Nederlands een `Authenticatie in twee stappen` noemen. Zie hier, als voorbeeld, de [Google 2FA pagina](https://support.google.com/accounts/answer/185839?hl=nl).

Reply:
Klonk beter in mijn hoofd. Nu ik het heb opgezocht, maakt het inderdaad niet uit. Ik kan die wel terugzetten.